avro-converter/src/main/java/io/confluent/connect/avro/AvroData.java

```diff
@@ -1445,7 +1446,7 @@ private Schema toConnectSchema(org.apache.avro.Schema schema, boolean forceOptio
       while (paramIt.hasNext()) {
         Map.Entry<String, JsonNode> field = paramIt.next();
         JsonNode jsonValue = field.getValue();
-        if (!jsonValue.isTextual()) {
+        if ((!field.getKey().endsWith(DOC_PROP) && !jsonValue.isTextual())) {
```

Comment:
@ewencp @pederpansen I think this only is happening for enums and this is how :

case ENUM:
        // enums are unwrapped to strings and the original enum is not preserved
        builder = SchemaBuilder.string();
        builder.parameter(CONNECT_ENUM_DOC_PROP, schema.getDoc()); // This is the place
  
and while reading the connect schema's connect.parameters we expect every thing to be not null which i think is a good assumptions provided we just let the DOC's go through